ha/core/event_manager/event_manager.py

```diff
@@ -37,15 +49,30 @@ def get_instance():
         return EventManager.__instance
 
     def __init__(self, singleton_check: bool = False):
-        """
-        Private Constructor.
-        """
+        '''
+           Private Constructor.
+           Make initialization work for Event Manager
+        '''
         if singleton_check is False:
             raise Exception("Please use EventManager.get_instance() to fetch singleton instance of class")
         if EventManager.__instance is None:
             EventManager.__instance = self
         else:
             raise Exception("EventManager is singleton class, use EventManager.get_instance().")
+        self._confstore = ConfigManager.get_confstore()
+
+    def _create_message_type(self, component):
```

Comment:
separate message bus code from this class 
https://github.com/Seagate/cortx-ha/pull/514/files ref util message_bus.py

Reply:
Added as a TODO for now.